sdk/storage/Azure.ResourceManager.Storage/tests/Helpers/StorageTestBase.cs

```diff
@@ -33,6 +33,7 @@ public class StorageTestBase : ManagementRecordedTestBase<StorageManagementTestE
         protected SubscriptionResource DefaultSubscription { get; private set; }
         protected StorageTestBase(bool isAsync) : base(isAsync)
         {
+            Assert.Ignore("Ignore storage tests until the API version 2022-09-01 is available.");
```

Comment:
The tests are skipped for now and will be re-recorded before this PR is merged in.